app/code/community/Businessfactory/Roihuntereasy/controllers/StoredetailsController.php

```diff
@@ -37,13 +37,13 @@ public function debugAction()
         $request = $this->getRequest();
         $response = $this->getResponse();
 
-        $response->setHeader('Content-type', 'application/json');
-        $response->setHeader('Access-Control-Allow-Origin', '*', true);
-        $response->setHeader('Access-Control-Allow-Methods', 'OPTIONS,GET,POST', true);
-        $response->setHeader('Access-Control-Max-Age', '60', true);
-        $response->setHeader('Access-Control-Allow-Headers', 'X-Authorization', true);
+        $response->setHeader("Content-type", "application/json");
```

Comment:
Move headers configuration to separate function - you have this block of code five times in this file.